spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -146,6 +146,33 @@ def func_map(data, mask, map_clusters):
     return beta[0], beta
 
 
+def func_median(data, mask=None, map_clusters=None):
```

Comment:
```suggestion
def func_median(data, mask, map_clusters=None):
```

Since we use `mask`, I think it isn't optional here.